resources/lib/rtvs.py

```diff
@@ -57,7 +57,7 @@
 START_LISTING = "<div class='calendar modal-body'>"
 END_LISTING = '</table>'
 LISTING_PAGER_RE = "<a class=\'prev calendarRoller' href=\'(?P<prevurl>[^\']+)\'.+?<a class=\'next calendarRoller\' href=\'(?P<nexturl>[^\']+)"
-LISTING_DATE_RE = '<div class=\'calendar-header\'>\s+<h6>(?P<date>[^<]+)</h6>'
+LISTING_DATE_RE = '<div class=\'calendar-header\'>\s+.*?<h6>(?P<date>[^<]+)</h6>'
```

Comment:
myslim, ze tu chyba este jeden line break
'<div class=\'calendar-header\'>\s+.*?\s+<h6>(?P<date>[^<]+)</h6>'

Reply:
funkcnost je odskusana. funguje to aj takto. line break netreba kvoli re.DOTALL. bude to teda fungovat v zapisoch
```
.*?
\s+.*?
\s+.*?\s+ 
```

Reply:
to mi uslo, vdaka za vysvetlenie ;)